src/asm_ostream.cpp

```
@@ -135,49 +95,59 @@ void print_goto(std::ostream& o, const value_t& value) {
+    out << "    node [shape = rectangle];\n";
+    for (const auto& label : cfg.labels()) {
+        out << "    \"" << label << "\"[xlabel=\"" << label << "\",label=\"";
+
+        for (const auto& pre : cfg.assertions_at(label)) {
+            out << "assert " << pre << "\\l";
         }
-        print_goto(os, cfg.get_node(bb.last_label()));
-        if (invariants) {
-            os << "\nPost-invariant: " << invariants->at(bb.last_label()).post << "\n";
+        out << cfg.instruction_at(label) << "\\l";
+
```

cppcheck (2.10-2)

[error] Syntax Error

(internalAstError)